.github/workflows/ci-integration.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        release: ['xenial', 'bionic', 'focal', 'jammy', 'kinetic']
+        release: ['xenial', 'bionic', 'focal', 'jammy', 'kinetic', 'lunar']
```

Comment:
This builds the package for lunar as part of CI. Should we also run the lunar integration tests as CI?

Reply:
Hm, I would say no, and the reason is: we do not run kinetic (:
I will extend your question instead of answering: should we run any non-lts as CI?

Reply:
Ahhh interesting!

I think we should, as we don't currently have any automated assurances that we aren't breaking non-lts during dev. Maybe let's only run the latest non-lts though?

Proposal: CI should cover lxd tests for:
* all supported LTS
* devel
  * if devel is LTS, then also the latest non-lts

Reply:
one point I have against it is that devel is going to change ALOT

Reply:
thinking twice after posting it:
- it will change a lot in LTSs, as services may change, not be ready yet, etc;

Reply:
excellent points

Once we have daily/weekly jenkins runs, maybe a devel run would work better there.

And here for CI we do:
* all supported LTS
* latest supported non-LTS

does that match what you're thinking?

Reply:
definitely. Will add kinetic integration here, and then we rotate it as new interim releases are, well, released.